sdk/spring/azure-spring-boot-samples/README.md

```diff
@@ -23,20 +23,22 @@ If you just want to run sample code with released version(not current), instead
 #### Clone project from master branch
 It means cloning the master branch, all the code, dependencies, and readme files are up to date.
 
-1. Clone code to local repo 
+1. Clone code to local repo.
 
    ```
    git clone https://github.com/Azure/azure-sdk-for-java.git
    ```
 
-2. Traverse to the root directory
-
-3. Build the whole product by executing the following command which may take several minutes
+2. Traverse to the root directory.
 
+3. Build the modules associated with the current module, such as `azure-spring-boot-sample-active-directory-webapp`.
    ```
-   mvn clean install -Dmaven.javadoc.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Drevapi.skip=true -Djacoco.skip=true -DskipTests -Dparallel-test-playback
+   mvn clean install -pl .\sdk\spring\azure-spring-boot-samples\azure-spring-boot-sample-active-directory-webapp\ -am
+   ```
+   If you want to build faster, you can append the following Maven parameters.
+   ```
+   -Dmaven.javadoc.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Drevapi.skip=true -Djacoco.skip=true -DskipTests -Dparallel-test-playback
```

Comment:
Maybe we can use this file instead?
https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/build-developing-version-artifacts-if-needed.md

Reply:
Refs: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-starter-active-directory#prerequisites

Reply:
Got it